Reneiski_Homework1/ReneiskiSelenium11/PageObjects/Popups/RegistrationFormPage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Common.WebElements;
+
+namespace ReneiskiSelenium11.PageObjects.Popups
+{
+    public class RegistrationFormPage
+    {
+        private MyWebElement FirstNameTextBox = new MyWebElement(By.XPath("//*[@id='firstName']"));
```

Comment:
in general, it's better to initialize elements as properties rather than variables (think why)
ex, private MyWebElement FirstNameTextBox => new MyWebElement(By.XPath("//*[@id='firstName']"));

may be don't correct everywhere but keep in mind

Reply:
Fixed